examples/tune_pytorch/flow_gpu.py

```diff
@@ -0,0 +1,79 @@
+from metaflow.cards import Image
+from gpu_profile import gpu_profile
+from metaflow import FlowSpec, step, card, kubernetes, pypi, metaflow_ray, current
+
+
+class RayTuneTorchGPU(FlowSpec):
+    epoch_size = 1024
+    test_size = 256
+    num_samples = 20
+
+    def _do_ray_job(self):
+        import ray
+        import numpy as np
+        import pandas as pd
+        from utils import run, plot
+        from matplotlib import pyplot as plt
+        from ray.air.config import ScalingConfig
+
+        ray.init()
+
+        search_space = {
+            "lr": ray.tune.sample_from(lambda _: 10 ** (-10 * np.random.rand())),
+            "momentum": ray.tune.uniform(0.1, 0.9),
+            "scaling_config": ScalingConfig(
+                use_gpu=True,
+                num_workers=4,
+                resources_per_worker={"CPU": 8, "GPU": 1},
+            ),
+        }
+
+        results_list = run(
+            search_space=search_space,
+            epoch_size=self.epoch_size,
+            test_size=self.test_size,
+            num_samples=self.num_samples,
+            smoke_test=True,
+            run_config_storage_path=current.ray_storage_path,
```

Comment:
Ray will be writing every experiment in a subdirectory. Can we find a way to pull that out of the task and save it to self so that users can use that later.

Reply:
Can I address this later? we can open it as a new issue..

Reply:
opened a new issue to handle it later: https://github.com/outerbounds/metaflow-ray/issues/19